src/include/rbd/librbd.h

```diff
@@ -1195,6 +1195,15 @@ CEPH_RBD_API ssize_t rbd_aio_compare_and_write(rbd_image_t image,
                                                rbd_completion_t c,
                                                uint64_t *mismatch_off,
                                                int op_flags);
+CEPH_RBD_API ssize_t rbd_aio_comparev_and_writev(rbd_image_t image,
```

Comment:
Two v's seem redundant -- we aren't going to have "comparev + write" and "compare + writev" variants:
```suggestion
CEPH_RBD_API ssize_t rbd_aio_compare_and_writev(rbd_image_t image,
```